lib/services/elasticsearch.js

```diff
@@ -631,43 +632,57 @@ class ElasticSearch extends Service {
     }
 
     esRequest.body = request.input.body.bulkData;
+    const kuzzleMetaCreated = {
+      author: userId,
+      createdAt: dateNow,
+      updatedAt: null,
+      updater: null,
+      active: true,
+      deletedAt: null
+    };
+    const kuzzleMetaUpdated = {
+      updater: userId,
+      updatedAt: dateNow
+    };
 
-    // set missing index & type if possible
-    esRequest.body.forEach(item => {
+    // set missing index & type if possible and add metadata
+    let lastAction; // NOSONAR
```

Comment:
What is the `// NOSONAR` for here?

Reply:
Without it Sonarqube is complaining about putting the declaration inside the `for(;;)`.  
In this case I want to keep the action occuring in the `n - 1` loop turn.  

This is the same annoying thing with the `let i;`.